### PR TITLE
feat: add signin items table

### PR DESCRIPTION
**Pull request by dacongda**
Fix: https://github.com/casdoor/casdoor/issues/2688

add signin items table

### CODE REVIEW

**Comment by dacongda** [comment]
find bug in login page

File: object/application.go [hunk 1]
```diff
@@ -40,6 +40,15 @@ type SignupItem struct {
 	Rule        string `json:"rule"`
 }
 
+type SigninItem struct {
+	Name        string `json:"name"`
+	Visible     bool   `json:"visible"`
+	Label       string `json:"label"`
+	Placeholder string `json:"placeholder"`
+	Rule        string `json:"rule"`
+	Custom      bool   `json:"custom"`
```
Comment (hsluoyz) [inline object/application.go:49]: IsCustom

File: web/src/table/SigninTable.js [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+            {name: "Logo", displayName: i18next.t("general:Logo")},
+            {name: "BackButton", displayName: i18next.t("login:Back button")},
+            {name: "LanguageSelect", displayName: i18next.t("general:Languages")},
+            {name: "Username", displayName: i18next.t("signup:Username")},
+            {name: "Password", displayName: i18next.t("general:Password")},
+            {name: "ThirdParty", displayName: i18next.t("general:Providers")},
+            {name: "Agreement", displayName: i18next.t("signup:Agreement")},
+            {name: "ForgetPassword", displayName: i18next.t("login:Forgot password?")},
+            {name: "LoginButton", displayName: i18next.t("login:Login button")},
+            {name: "Footer", displayName: i18next.t("general:Footer")},
```
Comment (hsluoyz) [inline web/src/table/SigninTable.js:107]: Footer -> Signup link

File: web/src/table/SigninTable.js [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+              value={text} onPressEnter={e => {
+                this.updateField(table, index, "name", e.target.value);
+              }} >
+            </Input>;
+          }
+
+          const items = [
+            {name: "MethodChoice", displayName: i18next.t("application:Signin methods")},
+            {name: "Logo", displayName: i18next.t("general:Logo")},
+            {name: "BackButton", displayName: i18next.t("login:Back button")},
```
Comment (hsluoyz) [inline web/src/table/SigninTable.js:99]: name should be the same as displayName

File: web/src/table/SigninTable.js [hunk 4]
```diff
@@ -0,0 +1,299 @@
+// Copyright 2021 The Casdoor Authors. All Rights Reserved.
```
Comment (hsluoyz) [inline web/src/table/SigninTable.js:1]: 2024

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**